packaging/installer/kickstart.sh

```diff
@@ -84,9 +84,15 @@ main() {
     exit 0
   fi
 
-  tmpdir="$(create_tmp_directory)"
-  progress "Using ${tmpdir} as a temporary directory."
-  cd "${tmpdir}" || fatal "Failed to change current working directory to ${tmpdir}." F000A
+  tempfoo=`basename -s .sh $0`
+  # mktemp behaves slightly differently on macOS but we ignore this fact...
+  wrkdir=`mktemp -q -d -t ${tempfoo}.XXXXX`
+    if [ $? -ne 0 ]; then
+      echo "$0: Can't create temp directory, exiting..."
+      exit 1
```

Comment:
This should be a call to `fatal` with an appropriate error code in addition to the message (_if_ we end up removing the `create_tmp_directory` code, it should reuse the F0400 error code being used there).